modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/utilities/index.js

```diff
@@ -16,6 +16,23 @@ import {fetch} from 'frontend-js-web';
 
 import createOdataFilter from './odata';
 
+export function getAcceptLanguageHeaderParam() {
+	const browserLang = navigator.language || navigator.userLanguage;
+	const themeLang = Liferay.ThemeDisplay.getLanguageId().replace('_', '-');
+
+	if (browserLang === themeLang) {
+		return browserLang;
+	}
+
+	return `${browserLang}, ${themeLang};q=0.8`;
+}
+
+const fetchHeaders = new Headers({
+	Accept: 'application/json',
+	'Accept-Language': getAcceptLanguageHeaderParam(),
```

Comment:
Just out of curiosity, it returns different data depending on the language, can this be a problem for other products that don't do the same?

Reply:
Indeed. We might want to consider adding this to our `fetch` in `frontend-js-web`

Reply:
In commerce we use these headers everywhere so I would say it's well tested but my experience is limited only to our domain :(